feature/interface/ip/ipv6_link_local/otg_tests/ipv6_link_local_test/ipv6_link_local_test.go

```diff
@@ -85,6 +87,14 @@ func TestIPv6LinkLocal(t *testing.T) {
 	otgSrcToDstFlow(t, top, ateSrc.IPv6, ateDst.IPv6, linkLocalFlowName)
 
 	ate.OTG().PushConfig(t, top)
+	// https://github.com/openconfig/featureprofiles/issues/3410
+	// Below code will be removed once ixia issue is fixed.
+	if slices.Contains(kneDeviceModelList, dut.Model()) {
+		ate.OTG().StartTraffic(t)
+		time.Sleep(15 * time.Second)
```

Comment:
Couple of comments here
1) Can we add a code to check if the traffic is started on Ixia and then follow step 2.
2) Instead of Sleeping for 15 seconds of time; We can make changes in the traffic profile to send fixed traffic with a duration of 15 or 20seconds and we would not have to stop the traffic in ln #9